src/cpp/src/continuous_batching/model_runner.hpp

```diff
@@ -48,6 +69,10 @@ class ModelRunner {
     // Input shape: [N, conversation length].
     // Output shape: [1, conversation length, hidden_size].
     EmbeddingsModel::Ptr m_embedding;
+    uint8_t m_hidden_state_flags = HS_NONE;
+    std::map<SequenceKey, HiddenStateRange> m_sequence_hidden_state_mapping;
+    // a container which use sequence group id and request id as key to store hidden states
+    std::map<size_t, ov::Tensor> m_initial_hidden_states; // shape: [N, seq_len, hidden_size]
```

Comment:
what is key of `size_t` type here? The comment looks incorrect "a container which use sequence group id and request id as key to store hidden states".
Please double-check

Reply:
I think we can use `unordered_map` because hash function for  standard type `size_t` is defined